### ISSUE TITLE
[MAS2.1.1] [Keyboard Navigation-File] File menu is not accessible using keyboard

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1802]
**Actual Result:**
File menu is not accessible using keyboard

**Expected Result:**
File menu should be accessible using keyboard

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Navigate to the File menu using keyboard  
3.Observe the issue 

**Note:** Same issue repro for below path 
Help->Get started with channels.
Help control is not accessible using keyboard 

**User Impact:**
Keyboard dependent users won't be able to use the controls which are not accessible using keyboard in any way.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
[MAS2.1.1_All headers.zip](https://github.com/microsoft/BotFramework-Emulator/files/3562137/MAS2.1.1_All.headers.zip)

**Comment by tonyanziano** [comment]
This is another piece of native OS UI, so I'm not sure what we can do about this other than writing a custom app menu.

**Comment by tonyanziano** [comment]
@Kumar2608 @awalia13 

If we make this menu accessible through a keyboard shortcut, such as "Alt + F" for **F**ile menu, would that be an acceptable solution or does it have to be accessible via "Tab" focus?

**Comment by Kumar2608** [comment]
Verified in latest bot emulator Version 4.5.2-78154 by using
 I have verified with "Alt+F" key still File menu is not accessible and better to provide tab focus also for keyboard dependent users  will help a lot .

**Comment by Kumar2608** [comment]
Verified in latest bot emulator Version 4.5.2-82320 Issue got fixed Hence we are closing this bug.
